kong/plugins/http-log/sender.lua

```diff
@@ -75,12 +75,13 @@ local function send(self, bodies)
     })
     if not res then
       log(ERR, "failed request to ", host, ":", tostring(port), ": ", err)
+      return false
+    else
```

Comment:
since the previous block already returns, the else branch is unnecessary

Reply:
Cleaned up!